### PR TITLE
Browser client compatibility

### PR DESCRIPTION
**Pull request by switters**
The browser client should probably send `text/plain` data, and the express server should likewise expect text data.

The reason is CORS definition of a [simple header](http://www.w3.org/TR/cors/#simple-header). A `Content-Type` header with a value of `application/json` is not considered a simple header, and therefore requires an additional preflight request. Sending two requests per log message is doubling the workload to no good end, so we should try to correct this.

### CODE REVIEW

**Comment by switters** [comment]
@lvivier - I tink zis mebe goot tyew goh. unless the new middleware sucks

**Comment by switters** [comment]
@wired8 @lvivier - review? I want to merge this in for working on #14.

File: lib/http.js [hunk 1]
```diff
@@ -3,10 +3,8 @@
  * Dependencies
  */
 
-var json = require('body-parser').json
```
Comment (lvivier) [inline lib/http.js]: might as well remove body-parser from package.json

**Comment by lvivier** [comment]
:+1:

**Pull request merged**